client/src/app/site/site.component.html

```diff
@@ -46,14 +46,15 @@
         <!-- navigation -->
         <mat-nav-list class='main-nav'>
             <span *ngFor="let entry of mainMenuService.entries">
-                <a [@navItemAnim]  *osPerms="entry.permission" mat-list-item (click)='toggleSideNav()'
-                    [routerLink]='entry.route' routerLinkActive='active' [routerLinkActiveOptions]="{exact: true}">
+                <a [@navItemAnim] *osPerms="entry.permission" mat-list-item (click)='toggleSideNav()' [routerLink]='entry.route'
+                    routerLinkActive='active' [routerLinkActiveOptions]="{exact: entry.route === '/' ? true: false}">
                     <fa-icon [icon]='entry.icon'></fa-icon>
-                    {{ entry.displayName | translate}}
+                    <span translate>{{entry.displayName}}</span>
```

Comment:
Just curious: What does this change do?

Reply:
fixing a regression.
`<span>` has been defined in the [theme](https://github.com/OpenSlides/OpenSlides/blob/OpenSlides-3/client/src/app/site/site.component.scss-theme.scss#L79). This makes the text blue (or rather $primary) again, if the router-link is active.